generator/sources/csharp/KalturaClientTester/KalturaUploadThread.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Threading;
+
+namespace Kaltura
+{
+    class KalturaUploadThread

```

Comment:
I don't think it fits as a facility class for a general purpose implementation.
specifically - because it uses Console.WriteLine and because it interacts with KalturaClientTester directly via
```
KalturaClientTester.workingThreads--;
```

This is more a sample implementation than a generic useful addition to the client lib itself.